frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -129,6 +128,15 @@ action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-fail=Sli
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.of-current=Branch <b>{0}</b> has been slid out but not deleted as it is currently checked out
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.with-delete=Branch <b>{0}</b> has been slid out and deleted
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.without-delete=Branch <b>{0}</b> has been slid out but not deleted
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git Machete Branch Deletion Suggestion
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.ok-text=_Delete Local Branch
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.no-text=_Keep Local Branch
```

Comment:
Hmmm okay there's an apparent bug here:

**Prerequisite**: slide out a branch, using `Keep Local Branch` + `Remember my choice`

**Action**: slide out another branch

**Expected**: the choice to keep the branch should be remembered

**Actual**: the dialog shows up, again asking whether I want to remember the choice

**Possible remedy**: possible when `Remember my choice` is selected, the config key should be set to `false` (rather than left intact); then, the dialog should only show up when the config key is empty/undefined, and NOT when it's set to `false`